error: custom attribute panicked
 --> tests/models/struct/named/u8/fail/validate.rs:2:1
  |
2 | #[model]
  | ^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: Error("expected identifier")
